addons/hr_recruitment_survey/models/survey_survey.py

```diff
@@ -14,7 +14,7 @@ def _compute_allowed_survey_types(self):
         super()._compute_allowed_survey_types()
```

Comment:
Don't really understand the commit message, it's mentioning a MOV, yet we are not moving anything, probably copy/pasta :smile:

Reply:
Made a slight modification to the commit message, I can assure you that this widget moved

```
$ git show --stat | grep radio_selection_field_with_filter.js
 addons/{survey/static/src/views/widgets/radio_selection_with_filter => web/static/src/views/fields/radio}/radio_selection_field_with_filter.js |  2 +-
```

Reply:
Ah indeed, I missed it, not sure if it's wise to move files in stable, but shouldn't be a big issue if we merge this quickly enough I assume.